src/XrdCeph/XrdCephOss.cc

```diff
@@ -46,6 +46,42 @@ XrdVERSIONINFO(XrdOssGetStorageSystem, XrdCephOss);
 XrdSysError XrdCephEroute(0);
 XrdOucTrace XrdCephTrace(&XrdCephEroute);
 
+bool allCharsAlnum(std::string candidate) {
+
+  bool retval = true;
+  for (auto it = candidate.cbegin(); it != candidate.cend(); ++it) {
+    if (!isalnum(*it)) {
+      retval = false;
+      break;
+    }
+  }
+  return retval;
+
+}
+
+string sanitizePath(const char *path) {
+
+  std::string sPath(path);
+  if (sPath.front() == '/' && sPath.length() > 1) {
```

Comment:
can there be a case where spath is an empty string? 
if so, is there anything in the function to be concerned with? Or, we can catch it at the start?